### ISSUE TITLE
Plugin Conflict: Query Monitor bar shows up in the pattern preview

### ISSUE THREAD

**Issue by bacoords** [studiopress/pattern-manager#193]
**Describe the bug**
If you're running the [Query Monitor](https://wordpress.org/plugins/query-monitor/) plugin, it shows up in the pattern preview.

**To Reproduce**
1. Enable the Query Monitor plugin.
2. Visit the Patterns page
3. See Query Monitor

**Screenshots**
<img width="401" alt="Screenshot 2023-06-06 at 3 12 26 PM" src="https://github.com/studiopress/pattern-manager/assets/6867360/fb0e246c-3fb4-4d86-877c-fd1cf649aec6">

**Version**
- 0.2.0

**Device and browser information**
 - OS: MacOS
 - Browser: Arc 0.106.1 & Chrome 114.0.5735.106

**Additional context**
You may be able to conditionally set the `QM_DISABLE` constant on your preview iframes.

**Comment by kienstra** [comment]
Hi @bacoords,
Thanks for opening this, and sorry for the delay.

How painful is this for you?  

I agree it's not good, but I'm hesitant to set `QM_DISABLE`. 

Thanks a lot for using PM and opening these issues.

**Comment by bacoords** [comment]
Yeah and as I did some digging after I posted this I could see that it would be hard to disable it conditionally from one plugin to another. 

The hacky workaround I have (if anyone else comes looking for it) is something along the lines of this in your `functions.php`: 

```php
function example() {
	echo '<style>#query-monitor-main.qm-peek,#query-monitor-main.qm-show {display:none !important;}</style>';
}
add_action( 'patternmanager_before_pattern_preview_wp_head', 'example' );
```

**Comment by kienstra** [comment]
Hi @bacoords,
Thanks a lot for adding the workaround.

It means a lot that you're using Pattern Manager, and took the time to open these issues.

**Comment by kienstra** [comment]
If it's alright, I'm closing this, as we won't fix it in this repo.
